netjsonconfig/backends/openwrt/converters/wireless.py

```diff
@@ -110,6 +114,16 @@ def __intermediate_encryption(self, wireless):
             uci['key'] = encryption['key']
         # add ciphers
         cipher = encryption.get('cipher')
+        if (
+            protocol == 'wpa3_personal'
+            or protocol == 'wpa2_personal_mixed'
+            or protocol == 'wpa3_enterprise'
+        ):
+            cipher = 'auto'
```

Comment:
> Why did you use auto here

Because it is converted to CCMP in the OpenWrt when WPA3 is used.
But indeed it is not clear, I fixed it.

> I think we can remove this part. The following code must be able to handle it:

We need this code when TKIP is specified.